.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,25 @@
+name: Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+- type: textarea
+  attributes:
+    label: Bug description
+    description: |
+      Please provide a clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Versions
+    description: |
+      Please include the following information if relevant:
+      1. Please attach your .toml config file and command line overrides.
```

Comment:
i would like to be more specific that we want them to give us everything we need to reproduce the issue ourselves.

one possibility is toml+cmdline+ versions.  Another might be 'runnable code snippet'